WatchApp Extension/Controllers/CarbEntryListController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  CarbEntryListController.swift
+//  WatchApp Extension
+//
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+import LoopCore
+import LoopKit
+import os.log
+import WatchKit
+
+class CarbEntryListController: WKInterfaceController, IdentifiableClass {
+    @IBOutlet private var table: WKInterfaceTable!
+
+    @IBOutlet private var cobLabel: WKInterfaceLabel!
+
+    @IBOutlet var totalLabel: WKInterfaceLabel!
+
+    @IBOutlet var headerGroup: WKInterfaceGroup!
+
+    private let log = OSLog(category: "CarbEntryListController")
+
+    private lazy var loopManager = ExtensionDelegate.shared().loopManager
+
+    private lazy var carbFormatter: QuantityFormatter = {
+        let formatter = QuantityFormatter()
+        formatter.numberFormatter.numberStyle = .none
+        return formatter
+    }()
+
+    private var observers: [Any] = [] {
+        didSet {
+            for observer in oldValue {
```

Comment:
What happens if the old observers and new observers overlap? I'm guessing this won't happen by the caller.

Reply:
addObserver returns a unique opaque object for each subscription, so they can't overlap.